entitlement management: add GET for accesspackageassignment (#17880)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/accesspackageassignment-get.md

ms.localizationpriority: medium
ms.prod: "governance"
doc_type: apiPageType
---
# Get accessPackageAssignment

Namespace: microsoft.graph

In [Azure AD entitlement management](../resources/entitlementmanagement-overview.md), retrieve the properties and relationships of an [accessPackageAssignment](../resources/accesspackageassignment.md) object.

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

| Permission type                        | Permissions (from least to most privileged) |
|:---------------------------------------|:--------------------------------------------|
| Delegated (work or school account)     | EntitlementManagement.Read.All, EntitlementManagement.ReadWrite.All  |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | EntitlementManagement.Read.All, EntitlementManagement.ReadWrite.All |

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /identityGovernance/entitlementManagement/assignments/{accessPackageAssignmentId}
```

## Optional query parameters

> This method supports the `$expand` OData query parameter to expand the relationships, to retrieve the `accessPackage`, `target` and `assignmentPolicy`.  For general information, see [OData query parameters](/graph/query-parameters).

Reviewer: Suggestion only if $select is supported
```suggestion
This method supports the `$expand` OData query parameter to expand the `accessPackage`, `target` and `assignmentPolicy`. It also supports the `$select` query parameter.  For general information, see [OData query parameters](/graph/query-parameters).
```
